corehq/apps/style/templates/style/includes/load_into_modal.html

```diff
@@ -1,7 +1,10 @@
 <script type="text/javascript">
+{% if request.use_requirejs %}
+    requirejs(["jquery", "bootstrap"], function($) {
```

Comment:
I'm leaning towards moving this code into the `callback` in the RequireJS config (and doing the same with other initialization-type code, like setting up `window.analytics`), for the sake of being strict about saying that a page should only have a single javascript entry point.